redaxo/src/core/lib/error_handler.php

```diff
@@ -62,7 +62,12 @@ public static function handleException($exception)
             $whoops = new \Whoops\Run();
             $whoops->writeToOutput(false);
             $whoops->allowQuit(false);
+
             $handler = new \Whoops\Handler\PrettyPageHandler();
+            if (ini_get('xdebug.file_link_format')) {
+                $handler->setEditor('xdebug');
```

Comment:
So kann man auch alle Editoren nutzen, man muss die URL nur in der php.ini in `xdebug.file_link_format` setzen (xdebug verlinkt dann auch entsprechend).

Bei mir hab ich das immer gesetzt, da symfony dann auch automatisch Dateien verlinkt.

Für mich reicht das so, so können wir uns die Einstellung sparen.